common/compression/unzip.cpp

```diff
@@ -953,8 +969,11 @@ Common::SharedArchiveContents unzOpenCurrentFile (unzFile file, const Common::CR
 		delete[] compressedBuffer;
 		return Common::SharedArchiveContents();
 	}
-
+#ifndef USE_ZLIB
 	uint32 crc32_data = crc.crcFast(uncompressedBuffer, s->cur_file_info.uncompressed_size);
+#else
+	uint32 crc32_data = crc32(0, uncompressedBuffer, s->cur_file_info.uncompressed_size);
```

Comment:
Is it worth having a common wrapper for this as well?

Reply:
I don't see easily how to fix this.
Maybe in gzio we could create a crc32 function which has a static CRC object?